lib/db.js

```diff
@@ -929,32 +928,6 @@ module.exports = (
     )
   }
 
-  // VERIFICATION REMINDERS
-
-  DB.prototype.createVerificationReminder = function (reminderData) {
```

Comment:
Worthwhile to do a PR to remove for auth-db-mysql?

Reply:
Yea, one day!